scripts/ci/a01/dockerfiles/py36/Dockerfile

```diff
@@ -9,8 +9,9 @@ RUN rm /usr/bin/python && ln /usr/local/bin/python3.6 /usr/bin/python && \
     apt-get update && apt-get install -y jq
 
 COPY docker_app/collect_tests.py /tmp/collect_tests.py
+COPY privates /tmp/privates
 
-RUN find /tmp/build -name '*.whl' | xargs pip install && \
+RUN find /tmp/build -name '*.whl' | xargs pip install -f /tmp/privates && \
```

Comment:
What's this for?

Reply:
In those case, you want to install a private package in the docker container. it is useful for testing pre-released features.